.github/workflows/build.yml

```diff
@@ -128,9 +92,20 @@ jobs:
       - name: Build Plugin
         run: ./gradlew buildPlugin
 
+      # Run intellij:verifyPlugin task.
+      - name: Verify Plugin
+        run: ./gradlew verifyPlugin
+
+      # Publish plugin to beta channel
+      - name: Publish Beta Plugin
+        env:
+          PUBLISH_TOKEN: ${{ secrets.JETBRAINS_TOKEN }}
```

Comment:
Where does this token live?

Reply:
Can't tell you that. It's a secret.

Ok so real answer, if you have Collaborator access you will see `Settings` -> `Secrets`. Github does not allow you to view the value from the UX and only Github Workflows triggered by people with Collaborator access will be able to access the value from within the workflow.